core/src/main/java/com/linecorp/armeria/client/SafeConnectionPoolListener.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2015 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.client;
+
+import java.net.InetSocketAddress;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linecorp.armeria.common.SessionProtocol;
+
+import io.netty.util.AttributeMap;
+
+/**
+ * Helper Handler delegate event to decorated  {@link ConnectionPoolListener}.
```

Comment:
Need to revise this Javadoc?

Reply:
Removed `SafeConnectionPoolListener` since just wrapping the notification code with `try-catch` seemed fine.